src/styles/_components/_carroussel.scss

```diff
@@ -0,0 +1,86 @@
+@mixin carroussel() {
+    width: 592px;
+    height: 340px;
+    overflow: hidden;
+    white-space: nowrap;
+    z-index: 2;
+
+    &__button {
+        position: absolute;
+        display: block;
+        width: 40px;
+        height: 40px;
+        top: 140px;
+        background-color: transparent;
+        border: 0;
+        outline: 0;
+        font-size: em(24px);
+        text-align: center;
+        z-index: 3;
+
+        &:hover {
+            cursor: pointer;
+            color: color('primary');
+        }
+
+        &--prev {
+            left: 0;
+        }
+
+        &--next {
+            right: 0;
+        }
+    }
+
+    &__container,
+    &__container__wrapper {
+        position: relative;
+        height: 100%;
+        transform: translate3d(0px,0px,0);
+        transition: transform $animation-duration $swiftOut; 
```

Comment:
I am not fully aware of the conventions followed here ^^' but it seems inconsistent to have some vars dasherized and some camelcased